models/tcp_route.go

```diff
@@ -42,47 +47,29 @@ func NewTcpRouteMappingWithModel(tcpMapping TcpRouteMapping) (TcpRouteMapping, e
 	}, nil
 }
 
-func NewTcpRouteMapping(routerGroupGuid string, externalPort uint16, hostIP string, hostPort uint16, ttl int) TcpRouteMapping {
-	return NewSniTcpRouteMapping(routerGroupGuid, externalPort, nil, hostIP, hostPort, ttl)
-}
-
-func NewSniTcpRouteMapping(routerGroupGuid string, externalPort uint16, sniHostname *string, hostIP string, hostPort uint16, ttl int) TcpRouteMapping {
-	mapping := TcpMappingEntity{
-		RouterGroupGuid: routerGroupGuid,
-		ExternalPort:    externalPort,
-		SniHostname:     sniHostname,
-		HostPort:        hostPort,
-		HostIP:          hostIP,
-		TTL:             &ttl,
-	}
-	return TcpRouteMapping{
-		TcpMappingEntity: mapping,
-	}
-}
-
-func NewTcpRouteMappingWithModificationTag(
+func NewTcpRouteMapping(
 	routerGroupGuid string,
 	externalPort uint16,
 	hostIP string,
 	hostPort uint16,
-	ttl int,
-	modTag ModificationTag,
-) TcpRouteMapping {
-	return NewSniTcpRouteMappingWithModificationTag(routerGroupGuid, externalPort, nil, hostIP, hostPort, ttl, modTag)
-}
-
-func NewSniTcpRouteMappingWithModificationTag(
-	routerGroupGuid string,
-	externalPort uint16,
+	hostTlsPort uint16,
```

Comment:
Looks like there is a difference when hostTlsPort is nil? Can this function accept the pointer?